microsoft-edge/toc.yml

```diff
@@ -261,6 +261,8 @@
               href: devtools-guide-chromium/media-panel/index.md
             - name: Memory
               items:
+                - name: Detached Elements
```

Comment:
All other items under the Memory TOC node start with a verb. We should do this here too. How about `Find detached elements`